blog/_src/posts/2024-08-14-racket-v8-14.md

```diff
@@ -8,17 +8,21 @@
 
 ----------------------------------------------------------------------
 
+We are pleased to announce Racket v8.14 is now available from [https://download.racket-lang.org/](https://download.racket-lang.org).
+
+## As of this release:
+
 - 64-bit ARM Windows is one of the standard pre-built downloads.
 
 - The `compiler/demod` language integrates demodularization into the
   program rather than an external tool. Using the language approach
   cooperates with other Racket tools and allows demodularization to be
   used on libraries. Additionally, demodularization handles submodules
   and can prune definitions, significantly shrinking the size of some
-  programs.
+  programs.  ['#lang compiler/demod'](https://docs.racket-lang.org/raco/demod.html#%28mod-path._compiler%2Fdemod%29)
 
 - The `contract-in` form enables contracts to be specified in an
-  importing module.
+  importing module. [8.6 Attaching Contracts to Values]([)](https://docs.racket-lang.org/reference/attaching-contracts-to-values.html#(part._attaching-contracts-to-values))
```

Comment:
This doesn't look like a valid syntax to create Markdown link to me...

Also, should the whole link be inside parentheses? It looks weird to suddenly have "8.6 Attaching Contracts to Values" out of nowhere.